Sources/Leaf/Node+Rendered.swift

```diff
@@ -8,7 +8,7 @@ extension Node {
         case let .number(number):
             return number.description.bytes
         case let .string(str):
-            return str.bytes
+            return str.htmlEscaped().bytes
```

Comment:
This shouldn't affect raw at all, this is only really in play for String variables. If you need unescaped variable, just pre-render your string to bytes.